### PR TITLE
Added documentation and screenshots to README

### PR DESCRIPTION
**Pull request by briantkatch**
I've added documentation and screenshots for the Ulysses theme to the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,19 @@
 
 > A dark theme for [Ulysses](http://ulyssesapp.com).
 
+![Ulysses for Mac](https://cloud.githubusercontent.com/assets/10686291/10810206/aa32a350-7dbb-11e5-8a17-88d0dd351438.png)
+![Ulysses for iOS](https://cloud.githubusercontent.com/assets/10686291/10810202/a1d48606-7dbb-11e5-885d-ba4b4d6f379d.PNG)
```
Comment (nesl247) [inline README.md]: Please submit screenshots to the `dracula.github.io` repository and then update these urls to match `dracula/template/README.md`.

File: README.md [hunk 2]
```diff
@@ -2,10 +2,19 @@
...
+
 ## Install
 
-Instructions coming soon.
+You can download Dracula from the [Ulysses Style Exchange][http://styles.ulyssesapp.com/bundle/Dracula/56319778b634a4410207d126].
```
Comment (nesl247) [inline README.md]: The url should be wrapped with `()` instead of `[]`.

**Comment by nesl247** [comment]
Thank you for the submission. You could please instead update the README.md to match `dracula/template/README.md` and do a PR against `dracula/dracula.github.io` and add the instructions there? We're currently trying to keep all of the instructions on the site until we update our site building system to use the READMEs of the individual themes.

**Comment by briantkatch** [comment]
Okay, I'll do that. Should I do anything with the original zenorocha/dracula-theme repository? It looks to me like the project has grown and you've moved to this GitHub organisation but I haven't seen any documentation on the original repository for the change, I just saw it in one of the pull requests there. Are you able to clarify as to what's happening with all the repositories?

**Comment by nesl247** [comment]
The old repository is still where it is while we finish up migrating Atom. Once that is done we will be moving it to the new organization. Nothing needs to be done there. Thanks for asking 😄 .

**Comment by briantkatch** [comment]
I've just looked at the template and other than the shadow on the screenshot and the "Instructions can be found at draculatheme.com/x" I don't see anything else to change. Currently there is no such page as draculatheme.com/ulysses, but if that's coming soon then I can still link to it. Also, I don't know how to take/edit a screenshot to have that shadow, could you tell me how?

**Comment by nesl247** [comment]
To take that screenshot, you do `CMD + SHIFT + 4` and then hit `Space` and then click the window you want to take a screenshot of.

The site isn't there yet, but that's what we actually need to do. I guess we can just leave the README be here until it's ready. In that case, can you please submit the screenshot to the site repo and update this README to link to that screenshot. You can see how in the template. Once the other PR is submitted and merged, I will review this one again.

Thanks!

**Comment by briantkatch** [comment]
The screenshot has been merged into the dracula/dracula.github.io but isn't live yet (It's a 404 when I access it but it may just be cached on GitHub's end). I've also updated the readme to fully conform to the template.

I've just looked at the directories for each theme under dracula/dracula.github.io/src/content, and looking at the `index.html` file for each theme, I'm not sure what the bit at the top between the `---`s is supposed the be or what I am supposed to put in that spot. Could I get some guidance?

**Comment by nesl247** [comment]
The top part of those files is the metadata for metalsmith, the static site generator. Don't worry about that for now anyways. We need to come up with an icon and everything for it before it could be merged in.

Reviewing your README now.

**Comment by nesl247** [comment]
Looks like we might have had a miscommunication, sorry about that. Please put your installation instructions back here in the README. The site will come later. Gotta get an icon and such as I mentioned, and that will take longer. Meanwhile it's better to have a README.

Sorry for the confusion.

**Comment by briantkatch** [comment]
I've made a new pull request on dracula/dracula.github.io for the Ulysses instructions. 

Have you considered just using the icons from the various apps/editors on the site? I don't know how that would look but it was a bit confusing for me that the icons are not related to the apps themselves. 

[Editorconfig](http://editorconfig.org) uses icons that are unique but still recognisable for each editor, if there was a designer who was willing to donate their time then maybe that would be a good approach. Another approach would be to simply have no icons but that may not be as appealing.

Edit: apologies, didn't mean to close.

**Comment by nesl247** [comment]
I'd have to let @zenorocha answer that one.

**Comment by nesl247** [comment]
@defectivebit I've gone ahead and merged this in. We've actually got a pending invite for you to be the maintainer of this theme. Please go ahead and accept that.

@zenorocha and I have been talking about how to handle the site and the repositories, so we'll be emailing all of the maintainers or contacting them on the repos or something soon. As a heads up, we're going to be moving all the screenshots to the repositories as `screenshot.png` in the root, and adding the installation instructions to the README. The site will eventually be built by the READMEs themselves. We'll be doing something to the site pretty soon to remove the duplication until that automation is completed.

If you'd like to get a head start on updating your repository, please feel free to do so. Sorry about the back and forth, we did a lot in such a short time and changed our minds frequently.

**Pull request merged**